bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusRollershutterModuleHandler.java

```diff
@@ -28,10 +42,37 @@
  */
 @NonNullByDefault
 public class NikobusRollershutterModuleHandler extends NikobusModuleHandler {
+    private final Logger logger = LoggerFactory.getLogger(NikobusRollershutterModuleHandler.class);
+    private final List<PositionEstimator> positionEstimators = new CopyOnWriteArrayList<>();
+
     public NikobusRollershutterModuleHandler(Thing thing) {
         super(thing);
     }
 
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        if (thing.getStatus() == ThingStatus.OFFLINE) {
+            return;
+        }
+
+        positionEstimators.clear();
+
+        try {
+            for (Channel channel : thing.getChannels()) {
+                PositionEstimatorConfig config = channel.getConfiguration().as(PositionEstimatorConfig.class);
+                if (config.delay >= 0 && config.duration > 0) {
+                    positionEstimators.add(new PositionEstimator(channel.getUID(), config));
+                }
+            }
+        } catch (RuntimeException e) {
```

Comment:
What runtime exceptions are you trying to catch here?

Reply:
You are right, removed. Also removed from `NikobusPushButtonHandler`